lib/ansible/modules/packaging/os/apt.py

```diff
@@ -1100,6 +1107,7 @@ def main():
                 force=force_yes,
                 dpkg_options=dpkg_options,
                 build_dep=state_builddep,
+                fixed=state_fixed,
```

Comment:
The tests are failing because `state_fixed` is undefined when `p['state'] != 'fixed'`.

Reply:
thanks for the check, that's actually pretty obvious looking at the code I've made!